RationaleMCP/0035/SpecificationText.md

```diff
@@ -50,7 +50,9 @@ Then, the `<context identifier>` behind the keyword `msgctxt` is the full name o
 After the `msgid` keyword the text string which shall be translated follows. It should contain the original string from the Modelica text representation. 
 Since in Modelica control sequences also start with a backslash and another backslash is used to use sequences literally or to hide double quotes, no change is required here. 
 But Modelica allows strings to go over more than one line, gettext does not.
-Therefore, Modelica strings must be split into consecutive strings at the line breaks, and the line breaks must be encoded with a "\n" at the end of each string, e.g.
+Therefore, line breaks in Modelica must be encoded with "\n".

```

Comment:
For me, this misses the point of how it really works.  In Modelica, both a literal newline and `\n` mean the same thing.  How this is encoded in the gettext files is up to the gettext specification, although as far as I can see all gettext format string variants actually use `\n` to encode a line break.

Reply:
It is saying that a line-break must be encoded with "\n" - since Modelica's literal line breaks cannot be handled.

Reply:
\n in Modelica strings are encoded as \n in the translation file strings. Literal line breaks are encoded as \n at the end of a string and another string follows.

Modelica string are not evaluated to put them into translation file, they are used as they are, with all control sequences and patterns without change. The translation has to reproduce the patterns. gettext itself does not know anything about it.